lrc_file/LrcFile.py

```diff
@@ -159,6 +198,7 @@ def __init__(
             "receiver_cpu_data",
         ],
         evaluated_cpu_metrics: list[str] = ["cpu"],
+        evaluated_tc_metrics: list[str] = ["time_taken", "num_rules"],
```

Comment:
Is `num_rules` evaluated? Isn't that just a configuration parameter?

Reply:
I believe this has been resolved by changing to `run_interval`